plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/BaseDeltaLakeConnectorSmokeTest.java

```diff
@@ -1777,4 +1927,40 @@ private Session broadcastJoinDistribution(boolean dynamicFilteringEnabled)
                 .setSystemProperty(ENABLE_DYNAMIC_FILTERING, Boolean.toString(dynamicFilteringEnabled))
                 .build();
     }
+
+    private String getTableLocation(String tableName)
+    {
+        Pattern locationPattern = Pattern.compile(".*location = '(.*?)'.*", Pattern.DOTALL);
+        Matcher m = locationPattern.matcher((String) computeActual("SHOW CREATE TABLE " + tableName).getOnlyValue());
+        if (m.find()) {
+            String location = m.group(1);
+            verify(!m.find(), "Unexpected second match");
+            return location;
+        }
+        throw new IllegalStateException("Location not found in SHOW CREATE TABLE result");
+    }
+
+    private String getTableComment(String tableName)
+    {
+        return (String) computeScalar(format(
+                "SELECT comment FROM system.metadata.table_comments WHERE catalog_name = '%s' AND schema_name = '%s' AND table_name = '%s'",
+                getSession().getCatalog().orElseThrow(),
+                SCHEMA,
+                tableName));
+    }
+
+    private String getColumnComment(String tableName, String columnName)
+    {
+        return (String) computeScalar(format(
+                "SELECT comment FROM information_schema.columns WHERE table_schema = '%s' AND table_name = '%s' AND column_name = '%s'",
+                SCHEMA,
+                tableName,
+                columnName));
+    }
+
+    private void dropTableFromMetastore(String tableName)
+    {
+        metastore.dropTable(SCHEMA, tableName, false);
+        assertThat(metastore.getTable(bucketUrl(), tableName)).isEmpty();
```

Comment:
redundant

Reply:
I think It's not redundant. We are not verifying table existence in test cases (except `testRegisterTable( )`)

Reply:
It's redundant (the behavior of `HiveMetastore#dropTable` should be tested in `HiveMetastore` test, not here) and also wrong (the first argument `getTable` should be schema name instead of `bucketUrl()`). I inlined this method.